src/Tmds.Ssh/SshClient.ChannelContext.cs

```diff
@@ -349,6 +349,27 @@ internal void DoDispose()
                 }
             }
 
+            public override async ValueTask SendChannelDataAsync(Packet packet, CancellationToken ct)
+            {
+                using var pkt = packet.Move();
+                long toSend = packet.PayloadLength - 9; // TODO: verify
```

Comment:
You can remove this TODO. `9` is the correct value. It is the offset of the data in the `SSH_MSG_CHANNEL_DATA` payload.